c2corg_api/models/user.py

```diff
@@ -103,6 +103,10 @@ class User(Base):
     feed_filter_activities = Column(
         ArrayOfEnum(enums.activity_type), nullable=False, server_default='{}')
 
+    # only show updates from followed users in the homepage feed
+    feed_followed_only = Column(
```

Comment:
Yes, that would be possible. For example by setting a parameters when requesting the feed (e.g. `/personal-feed?filter=0`).